### PR TITLE
Model for ultrasound simulation

### PR DESCRIPTION
**Pull request by Rakshith2597**
Submitting model for breast ultrasound simulation.

This is part of the project MIRIAD: Many Incarnations of Screening of Radiology for High Throughput Disease Screening via Multiple Instance Reinforcement Learning with Adversarial Deep Neural Networks, sponsored by INTEL TECHNOLOGY INDIA PVT. LTD.

Principal Investigators:

Dr Debdoot Sheet (PI), Dr Nirmalya Ghosh (Co-PI)
Department of Electrical Engineering
Indian Institute of Technology Kharagpur

Dr Ramanathan Sethuraman (Co-PI)
Intel Technology India Pvt. Ltd.

### CODE REVIEW

File: models/public/breast_ultrasound_simulation/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Results
+
+![](./media/cart.jpg)
+
+
+## Datasets
+The network is trained on [IVUS](http://www.cvc.uab.es/IVUSchallenge2011/dataset.html) dataset. Data augmentation is applied as explain in [1,2,3] and images are processed by [stage 0](https://in.mathworks.com/matlabcentral/fileexchange/34199-pseudo-b-mode-ultrasound-image-simulator). We have directly provided the processed dataset [here]([link](https://drive.google.com/drive/folders/1d4iu2OHxSaORK4mPXqb0Wy2ivXbXZIdI?usp=sharing)). 
+
+Segmentation masks from [INbreast](http://medicalresearch.inescporto.pt/breastresearch/index.php/Get_INbreast_Database) dataset is used to generate stage 0 simulation results for breast ultrasound.
```
Comment (vladimir-dudnik) [inline models/public/breast_ultrasound_simulation/README.md:21]: can't access specified link - The requested URL was not found on this server.

File: models/public/breast_ultrasound_simulation/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+<p align="center"><img src="./media/graphic.jpg" alt="drawing" width="500"/></p>
+
+## Results
+
+![](./media/cart.jpg)
+
+
+## Datasets
+The network is trained on [IVUS](http://www.cvc.uab.es/IVUSchallenge2011/dataset.html) dataset. Data augmentation is applied as explain in [1,2,3] and images are processed by [stage 0](https://in.mathworks.com/matlabcentral/fileexchange/34199-pseudo-b-mode-ultrasound-image-simulator). We have directly provided the processed dataset [here]([link](https://drive.google.com/drive/folders/1d4iu2OHxSaORK4mPXqb0Wy2ivXbXZIdI?usp=sharing)).
```
Comment (vladimir-dudnik) [inline models/public/breast_ultrasound_simulation/README.md:19]: can't fine dataset license information by this link

File: models/public/breast_ultrasound_simulation/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+Department of Electrical Engineering,</br>
+Indian Institute of Technology Kharagpur</br>
+email: debdoot@ee.iitkgp.ac.in, nirmalya@ee.iitkgp.ac.in
+
+<a href="https://www.linkedin.com/in/ramanathan-sethuraman-27a12aba/">Dr Ramanathan Sethuraman</a>,</br>
+Intel Technology India Pvt. Ltd.</br>
+email: ramanathan.sethuraman@intel.com
+
+## Publications
```
Comment (vladimir-dudnik) [inline models/public/breast_ultrasound_simulation/README.md:70]: Legal Information section is missed. It should specify model license

File: models/public/breast_ultrasound_simulation/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+## **Specification and Performance**
+
+
+| Type | Simulation |
+| --- | --- |
+| Source Framework | Pytorch (converted to ONNX) |
+| FLOPs | 34,027 |
+| params | 33,640 |
+
```
Comment (vladimir-dudnik) [inline models/public/breast_ultrasound_simulation/README.md:32]: Accuracy section is missed. You need specify what accuracy model should demonstrate on validation set, what accuracy metric is measured, and how it is computed

File: models/public/breast_ultrasound_simulation/README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+## **Input and Output**
```
Comment (vladimir-dudnik) [inline models/public/breast_ultrasound_simulation/README.md:33]: There should be separate sections, Input and Output, to describe expected format of model's input and output blobs

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]
we expect contributor will provide all required legal information (model and dataset license, implement accuracy validation support in accuracy-checker and provide corresponding configuration file, implement demo to showcase basic model work)

**Comment by vladimir-dudnik** [comment]
cc @yssemaev

**Comment by yssemaev** [comment]
Hi,
The link is not working for me: http://www.cvc.uab.es/IVUSchallenge2011/dataset.htm
